Jenkinsfile

```diff
@@ -33,6 +33,7 @@ pipeline {
 
   post {
     always {
+      sh 'sudo chown -R jenkins:jenkins .'  // bad docker mount creates unreadable files TODO fix this
```

Comment:
I think we're trying to move away from having `jenkins` have passwordless `sudo`. Maybe you could do something like this, instead: https://github.com/conjurinc/appliance/blob/master/Jenkinsfile#L62